app/assets/javascripts/components/school_search.js.erb

```diff
@@ -82,15 +82,13 @@ document.addEventListener("DOMContentLoaded", function() {
       suggestion: function(id) {
         var school = findSchool(id);
 
-        var suggestion =
-          '<label class="govuk-label govuk-label--s">' +
-          school.name +
-          "</label>" +
-          '<span class="govuk-hint">' +
-          school.address +
-          "</span>";
-
-        if (school.closeDate) suggestion += '<span class="govuk-hint">' + "Closed on " + school.closeDate + "</span>";
+        var suggestion = '<table><tr><td><label class="govuk-label govuk-label--s">' + school.name + "</label></td>";
+
+        if (school.closeDate)
+          suggestion += '<td><span class="govuk-hint">' + "Closed on " + school.closeDate + "</span></td>";
+
+        suggestion +=
+          '</tr><tr><td colspan="2"><span class="govuk-hint">' + school.address + "</span></td><tr></table>";
```

Comment:
Isn't it bad for accessibility to use tables when you're not actually displaying tabular data? Are they just for layout?

Reply:
Ah, that's a very good point...

Reply:
Ok no worries, will swap to divs and flex or inline it.